### DISCUSSION TITLE
Is there a way to have tab like feature (like browser) for multiple pdfs?

### DISCUSSION THREAD

**Discussion by MoreRational** [ahrm/sioyek#1115] [Q&A]
Navigation between pdfs would be easier. Sometimes we need to refer to multiple pdfs.

**Comment by ahrm** [comment]
In the development branch we have a tabs feature. It is not exactly like browsers though (there is no tab bar on top of the screen, you open the tabs menu by issuing a command)

**Comment by micangl** [comment]
Would it be possible to implement a tab bar as a plugin?

**Reply by ahrm** [reply]
No, the whole philosophy of sioyek was to remove the toolbars and distractions as much as possible. I think anything that you would want to do with a tab bar you can already do better with the current tools that we have.
